### PR TITLE
updating the center most member

### PR DESCRIPTION
**Pull request by Benjamindodgson**

### CODE REVIEW

File: iOS/Collection View Management /DiffableCollectionViewController.swift [hunk 1]
```diff
@@ -85,6 +84,7 @@ class DiffableCollectionViewController<SectionType: Hashable,
         }
 
         self.collectionView.animationView.stop()
+        self.handleDataBeingLoaded()
```
Comment (Benjamindodgson) [inline iOS/Collection View Management /DiffableCollectionViewController.swift:87]: Now called whenever the data is reloaded. I think before we only wanted it called once, but now we have reconfigure so.

Comment (MartinJibber) [reply iOS/Collection View Management /DiffableCollectionViewController.swift:87]: Makes sense

File: iOS/Flows/Conversation/Cells/ConversationMessagesCell.swift [hunk 2]
```diff
@@ -209,3 +213,15 @@ extension ConversationMessagesCell: UICollectionViewDelegate {
...
+extension ConversationMessagesCell: TimeMachineCollectionViewLayoutDelegate {
+    func timeMachineCollectionViewLayout(_ layout: TimeMachineCollectionViewLayout, updatedFrontmostItemAt indexPath: IndexPath) {
+        guard indexPath.section == 0, let item = self.dataSource.itemIdentifier(for: indexPath) else { return }
+        switch item {
+        case .message(cid: let cid, messageID: let messageID):
+            self.incomingTopMostMessage = ChatClient.shared.message(cid: cid, id: messageID)
+        case .loadMore(_):
+            break
+        }
+    }
```
Comment (MartinJibber) [inline iOS/Flows/Conversation/Cells/ConversationMessagesCell.swift:250]: Looks good

File: iOS/Flows/Conversation/Conversation Header/MembersViewController.swift [hunk 3]
```diff
@@ -74,6 +76,16 @@ class MembersViewController: DiffableCollectionViewController<MembersCollectionV
             }
         }).store(in: &self.cancellables)
     }
+    
+    override func handleDataBeingLoaded() {
+        super.handleDataBeingLoaded()
+        
+        /// This is needed because sometimes the author get set before the datasource
```
Comment (MartinJibber) [inline iOS/Flows/Conversation/Conversation Header/MembersViewController.swift:83]: Triple slash should be used for documentation. This is an internal comment so double slash should be used.

**Review by MartinJibber — APPROVED** [review]
Looks good. I have a minor comment you can address if you want.

**Pull request merged**